frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx

```diff
@@ -76,13 +77,14 @@ const ExternalSystemSelection: React.FC<ExternalSystemSelectionProps> = ({
         aria-label={t('ceph-storage-plugin~Select external system from list')}
         value={selectedStorage}
         id="storage-platform-name"
-        className="odf-backing-storage__selection--width"
+        className="odf-backing-storage__selection--width  odf-backing-storage__selection--spacer"
         onChange={handleSelection}
       >
         {selectOptions.map(({ displayName, model: { kind } }) => (
           <FormSelectOption key={kind} value={kind} label={displayName} />
         ))}
       </FormSelect>
+      {selectedStorage === IBMFlashSystemModel.kind && <DevPreviewBadge />}
```

Comment:
```suggestion
      {selectedStorage !== OCServiceModel.kind && <DevPreviewBadge />}
```
We should check for the OCS service model kind , calling external vendor variables shoudl be avoided, so that when we detach those , it wont pain odf code.

Reply:
We are using those models throughout the code. And only IBM is in dev preview. If other vendors might come they might not be dev preview. The dependency on the external vendor variable unfortunately is already in too many places.

Reply:
These models are not used throughout. Its only used in IBM based folder.
New external vendor are not coming in any soon.

Reply:
It's however used in our repository so it really doesn't make a difference. As long as it's part of our repo this is equivalent to it.

Reply:
I don't find it equivalent. We need to make change and remove references when we separate.
This is totally not right IMO. Not blocking PR but I dont approve such hard code references